app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/ContributorsActivity.java

```diff
@@ -51,8 +54,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.item_repository:
-                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/DroidKaigi/conference-app-2017"));
-                startActivity(intent);
+                navigator.navigateToWebPage("https://github.com/DroidKaigi/conference-app-2017");
```

Comment:
@k-kagurazaka Thanks! As you said, it's not perfect but good 👍 

@ogapants Can you change like this? 🙏 
1. Create new `onClickRepositoryMenu()` method to `ContributorsViewModel`
2. Inject it to `ContributorsFragment` class
3. Call the method here